frontend/src/components/MatchingPairs/MatchingPairs.jsx

```diff
@@ -116,7 +116,9 @@ const MatchingPairs = ({
                 currentCard.noevents = true;
                 currentCard.seen = true;
                 currentCard.boardposition = parseInt(index) + 1;
-                currentCard.timestamp = performance.now();                
+                currentCard.timestamp = performance.now();
+                // reset response interval in case this card has a value from a previous turn
+                currentCard.response_interval_ms = '';
```

Comment:
Wouldn't a `null` or `0` value be better though since `performance.now()` returns a `number`?